src/app.coffee

```diff
@@ -1,4 +1,5 @@
 async = require 'async'
+FilterManager = require 'cozy-device-sdk'
```

Comment:
I don't like class named in SomethingManager. Cf https://blog.codinghorror.com/i-shall-call-it-somethingmanager/

And why filter instead of views, as it is the terminology in CouchDB world?